Movie Challenge/Movie Challenge/Controllers/Favorite/FavoriteViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  NewFavoriteViewController.swift
+//  Movie Challenge
+//
+//  Created by Igor Gomes Arantes on 11/09/2018.
+//  Copyright © 2018 igor gomes arantes. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class FavoriteViewController : UITableViewController{
+    
+    @IBOutlet var favoriteTableView: UITableView!
+    
+    var categoryList = [CategoryDTO]()
+    
+    override func viewWillAppear(_ animated: Bool) {
+        do{
```

Comment:
You forgot of call the "super.viewWillAppear" method, this could cause serious problems becouse kill the view life cicle.